AutomationHelpers/scripts/install.ps1

```diff
@@ -0,0 +1,27 @@
+# Copyright � 2018 Ranorex All rights reserved
+
+param($installPath, $toolsPath, $package, $project)
+
+Write-Host 'Started adding constant for current Ranorex version to compilation symbols...'
+
+$rxVersion = $project.Properties.Item("RanorexVersion")
+$rxVersion = $rxVersion.Value -replace '\.'
+if (!$rxVersion)
+{
+    Write-Warning('Could not find Ranorex version property in project.')
+    exit
+}
+$rxVersion = "RX$rxVersion"
+
+$defineConstants = $project.Properties.Item("DefineConstants")
+[array]$symbols = $defineConstants.Value.Split(';')
+Write-Host "Found: $symbols"
+$symbols = $symbols | Where {$_[0] -ne 'R' -and $_[1] -ne 'X'}
```

Comment:
Probably also check length == 4

Reply:
You see no chance for having Ranorex 10.0?

Reply:
Well, 10 maybe, 100 probably not ;-)